enhancements/network/static-ip-addresses-vsphere.md

```diff
@@ -0,0 +1,560 @@
+99_openshift-cluster-api_worker-machineset-1.yaml
+99_openshift-cluster-api_worker-machineset-2.yaml
+99_openshift-cluster-api_worker-machineset-3.yaml
+99_openshift-cluster-api_worker-user-data-secret.yaml
+99_openshift-machine-api_address-ngirard-dev-lbfxg-master-0-claim-0-0.yaml
+99_openshift-machine-api_address-ngirard-dev-lbfxg-master-1-claim-0-0.yaml
+99_openshift-machine-api_address-ngirard-dev-lbfxg-master-2-claim-0-0.yaml
+99_openshift-machine-api_address-ngirard-dev-lbfxg-worker-0-claim-0-0.yaml
+99_openshift-machine-api_address-ngirard-dev-lbfxg-worker-1-claim-0-0.yaml
+99_openshift-machine-api_address-ngirard-dev-lbfxg-worker-2-claim-0-0.yaml
+99_openshift-machine-api_claim-ngirard-dev-lbfxg-master-0-claim-0-0.yaml
+99_openshift-machine-api_claim-ngirard-dev-lbfxg-master-1-claim-0-0.yaml
+99_openshift-machine-api_claim-ngirard-dev-lbfxg-master-2-claim-0-0.yaml
+99_openshift-machine-api_claim-ngirard-dev-lbfxg-worker-0-claim-0-0.yaml
+99_openshift-machine-api_claim-ngirard-dev-lbfxg-worker-1-claim-0-0.yaml
+99_openshift-machine-api_claim-ngirard-dev-lbfxg-worker-2-claim-0-0.yaml
+99_openshift-machine-api_master-control-plane-machine-set.yaml
+99_openshift-machineconfig_99-master-ssh.yaml
+99_openshift-machineconfig_99-worker-ssh.yaml
+99_role-cloud-creds-secret-reader.yaml
+openshift-install-manifests.yaml
+```
+
+Using master-0 as the target example, we can see how the machine, IPAddressClaim and IPAddress are configured as follows:
+
+Machine:
+```yaml
+apiVersion: machine.openshift.io/v1beta1
+kind: Machine
+metadata:
+  creationTimestamp: null
+  labels:
+    machine.openshift.io/cluster-api-cluster: ngirard-dev-fhshn
+    machine.openshift.io/cluster-api-machine-role: master
+    machine.openshift.io/cluster-api-machine-type: master
+  name: ngirard-dev-fhshn-master-0
+  namespace: openshift-machine-api
+spec:
+  lifecycleHooks: {}
+  metadata: {}
+  providerSpec:
+    value:
+      apiVersion: machine.openshift.io/v1beta1
+      credentialsSecret:
+        name: vsphere-cloud-credentials
+      diskGiB: 100
+      kind: VSphereMachineProviderSpec
+      memoryMiB: 16384
+      metadata:
+        creationTimestamp: null
+      network:
+        devices:
+          - addressesFromPools:
+              - group: installer.openshift.io
+                name: default-0
+                resource: IPPool
+            nameservers:
+              - 8.8.8.8
+            networkName: ocp-ci-seg-13
+      numCPUs: 4
+      numCoresPerSocket: 2
+      snapshot: ""
+      template: ngirard-dev-fhshn-rhcos-us-east-us-east-1a
+      userDataSecret:
+        name: master-user-data
+      workspace:
+        datacenter: IBMCloud
+        datastore: /IBMCloud/datastore/mdcnc-ds-1
+        folder: /IBMCloud/vm/ngirard-dev-fhshn
+        resourcePool: /IBMCloud/host/vcs-mdcnc-workload-1//Resources
+        server: ibmvcenter.vmc-ci.devcluster.openshift.com
+status: {}
+```
+
+IPAddressClaim:
+```yaml
+apiVersion: ipam.cluster.x-k8s.io/v1alpha1
```

Comment:
Has the ipam stuff moved out of alpha yet?

Reply:
Yes.  It is v1beta1 now.  We are starting to pull these in.  I'll update this example to reflect that.